### PR TITLE
[px/pod] add widget to display node the pod runs on

### PR DESCRIPTION
**Pull request by htroisi**
image

### CODE REVIEW

File: pxl_scripts/px/pod/pod.pxl [hunk 1]
```diff
@@ -40,6 +39,21 @@ def containers(start_time: str, pod: px.Pod):
...
+
+    Args:
+    @start_time: The timestamp of data to start at.
+    @pod: The name of the pod to filter on.
+
+    '''
+    df = px.DataFrame(table='process_stats', start_time=start_time)
+    df = df[df.ctx['pod'] == pod]
+    df.name = df.ctx['node_name']
+    df = df.groupby(['name']).agg()
```
Comment (nserrino) [inline pxl_scripts/px/pod/pod.pxl:53]: nit: you can omit the bracket and just say 'name' when the list only has one element

**Review by nserrino — APPROVED** [review]

**Comment by philkuz** [comment]
Change the git title to px/pod rather than px/node.


Also I would also add info abt the service that runs the pod. Maybe something like Pod Metadata?

**Review by philkuz — CHANGES_REQUESTED** [review]
rename pr and add more metadata to the table. We can also discuss over slack

**Comment by htroisi** [comment]
image

**Comment by zasgar** [comment]
I think this is ok for now, but we should probably have a custom widget type to make the formatting better.

**Review by zasgar — APPROVED** [review]

**Pull request merged**